source/fundamentals/data-formats/guid-serialization.txt

```diff
@@ -0,0 +1,173 @@
+.. _csharp-guids:
+
+==================
+GUID Serialization
+==================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to serialize **globally unique identifiers**  
+(`GUIDs <https://learn.microsoft.com/en-us/dynamicsax-2012/developer/guids>`__),
+also known as **universally unique identifiers** (UUIDs).
+
+A GUID is a 16-byte integer that you can use as a unique ID for a MongoDB document. 
+Originally, GUIDs in MongoDB were represented as ``BsonBinaryData`` values of subtype 3. 
+Subtype 3 did not standardize the byte order during serialization, which led to 
+inconsistent serialization across MongoDB drivers. 
+To standardize the byte order and ensure consistent serialization across drivers, we 
+created ``BsonBinaryData`` subtype 4.
+
+.. tip::
+   
+   Use ``BsonBinaryData`` subtype 4 for all new GUIDs. 
```

Comment:
It's presented in the docs as a "recommendation": "The recommendation is that all Guids in a collection be represented uniformly using the standard BsonBinaryData subtype 4." I shortened that to just encouraging readers to use subtype 4, since it isn't strictly required. I'll make it a note and see what they say during tech review; maybe there are valid reasons to still use subtype 3 (though I can't imagine what they'd be).

Reply:
Sounds good. Based on the way the opening paragraphs are worded it sounds like it's a pretty strong recommendation which is what makes me lean more towards a note. I'm down to change it if needed though